src/config.rs

```diff
@@ -133,6 +138,18 @@ impl Config {
             config.excerpt_separator = excerpt_separator.to_owned();
         };
 
+        #[cfg(all(feature="syntax-highlight", not(windows)))]
+        {
+            if let Some(theme) = yaml["syntax-highlight"]["theme"].as_str() {
+                if has_theme(theme) {
+                    config.theme = theme.to_owned();
+                } else {
+                    warn!("Theme named '{}' is not loaded. Using default 'base16-ocean.dark'.",
+                          theme);
+                }
+            };
+        }
+
```

Comment:
Agreed about not giving windows users false expectations. There is a tension between conveying accurate information and allowing a linux-deployed blog be written on Windows when the supported features are different.

While I've not yet looked at the diff, I wonder if I was unclear about my suggestion. If syntax highlighting is unsupported, that is when you ignore the value. That allows someone to write their blog on windows with successful builds (I consider WSL a cop out). They then get their feedback on themes when the build fails or succeeds on Travis. Logging a message when a theme is set on a platform with no syntax highlighting seems like a good compromise if a windows developer's expectations aren't met to help them realize what is going on.